salt/metalk8s/salt/master/files/salt-master-manifest.yaml.j2

```diff
@@ -78,6 +78,8 @@ spec:
         - name: kubernetes-pki
           mountPath: '/etc/kubernetes/pki'
           readOnly: true
+        - name: salt-master-log
+          mountPath: '/var/log/salt/master'
```

Comment:
Isn't salt-master logging to `stdout`/`stderr` when running in foreground mode?

> It makes easier to read and debug salt-master

Isn't `kubectl logs ...` sufficient?

Reply:
I've removed the commit.

Reply:
One could argue we already have Loki for that, but Loki could be down.
Moreover, from my PoV, it's much simpler to have the log directly accessible on your host (to grep, etc.), rather than having to go on a UI.
But I agree, that's not 100% necessary.

Reply:
You can `grep` on the output of `kubectl logs` :wink:
If we start bypassing the container logging approach to get them on the host, we'll have to maintain our own rotation, monitor the space it takes, etc.
And for the log history, we have Loki indeed :) (which should be usable without a UI FWIW)